File: test/react-native/features/fixtures/ios-module/Scenarios/NativeStackHandledScenario.m
#import "NativeStackHandledScenario.h"

@implementation NativeStackHandledScenario

- (void)run: (RCTPromiseResolveBlock)resolve
     reject:(RCTPromiseRejectBlock)reject {
  reject(@"NativeError", @"NativeStackHandledScenario", [NSError errorWithDomain:@"com.example" code:408 userInfo:nil]);
}

@end

File: test/react-native/features/fixtures/scenario-launcher/ios/Scenarios/NativeStackHandledScenario.swift
//
//  NativeStackHandledScenario.swift
//  reactnative
//
//  Created by Nick Dowell on 02/12/2020.
//

class NativeStackHandledScenario: Scenario {
  
  override func run(_ resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
    reject("NativeError", "NativeStackHandledScenario", NSError(domain:"com.example", code:408))
  }
}
